generate_toolchain.sh

```diff
@@ -93,390 +126,27 @@ tar xzf /opt/cmake-3.22.1-linux-x86_64.tar.gz --strip-components=1 --exclude='*/
-    /usr/include/x86_64-linux-gnu/sys/param.h \
-    /usr/include/x86_64-linux-gnu/sys/poll.h \
-    /usr/include/x86_64-linux-gnu/sys/prctl.h \
-    /usr/include/x86_64-linux-gnu/sys/procfs.h \
-    /usr/include/x86_64-linux-gnu/sys/ptrace.h \
-    /usr/include/x86_64-linux-gnu/sys/random.h \
-    /usr/include/x86_64-linux-gnu/sys/resource.h \
-    /usr/include/x86_64-linux-gnu/sys/select.h \
-    /usr/include/x86_64-linux-gnu/sys/sem.h \
-    /usr/include/x86_64-linux-gnu/sys/sendfile.h \
-    /usr/include/x86_64-linux-gnu/sys/shm.h \
-    /usr/include/x86_64-linux-gnu/sys/signal.h \
-    /usr/include/x86_64-linux-gnu/sys/socket.h \
-    /usr/include/x86_64-linux-gnu/sys/stat.h \
-    /usr/include/x86_64-linux-gnu/sys/statfs.h \
-    /usr/include/x86_64-linux-gnu/sys/statvfs.h \
-    /usr/include/x86_64-linux-gnu/sys/syscall.h \
-    /usr/include/x86_64-linux-gnu/sys/sysinfo.h \
-    /usr/include/x86_64-linux-gnu/sys/syslog.h \
-    /usr/include/x86_64-linux-gnu/sys/sysmacros.h \
-    /usr/include/x86_64-linux-gnu/sys/time.h \
-    /usr/include/x86_64-linux-gnu/sys/timerfd.h \
-    /usr/include/x86_64-linux-gnu/sys/times.h \
-    /usr/include/x86_64-linux-gnu/sys/ttydefaults.h \
-    /usr/include/x86_64-linux-gnu/sys/types.h \
-    /usr/include/x86_64-linux-gnu/sys/ucontext.h \
-    /usr/include/x86_64-linux-gnu/sys/uio.h \
-    /usr/include/x86_64-linux-gnu/sys/un.h \
-    /usr/include/x86_64-linux-gnu/sys/user.h \
-    /usr/include/x86_64-linux-gnu/sys/utsname.h \
-    /usr/include/x86_64-linux-gnu/sys/vfs.h \
-    /usr/include/x86_64-linux-gnu/sys/wait.h \
-    toolchain
-
-# Additional packages
-cp -r --parents \
-    /usr/include/zlib.h \
-    /usr/include/zconf.h \
-    /usr/include/openssl \
-    /usr/include/x86_64-linux-gnu/openssl \
-    toolchain
-
-# compiler-rt
-cp -r --parents \
-    /usr/include/crypt.h \
-    /usr/include/fstab.h \
-    /usr/include/mqueue.h \
-    /usr/include/net/if_ppp.h \
-    /usr/include/net/route.h \
-    /usr/include/netax25/ax25.h \
-    /usr/include/netinet/ether.h \
-    /usr/include/net/ppp_defs.h \
-    /usr/include/netinet/if_ether.h \
-    /usr/include/netipx/ipx.h \
-    /usr/include/netrom/netrom.h \
-    /usr/include/obstack.h \
-    /usr/include/scsi/scsi.h \
-    /usr/include/utmp.h \
-    /usr/include/utmpx.h \
-    /usr/include/x86_64-linux-gnu/bits/mqueue.h \
-    /usr/include/x86_64-linux-gnu/bits/mqueue2.h \
-    /usr/include/x86_64-linux-gnu/bits/msq.h \
-    /usr/include/x86_64-linux-gnu/bits/utmp.h \
-    /usr/include/x86_64-linux-gnu/bits/utmpx.h \
-    /usr/include/x86_64-linux-gnu/sys/kd.h \
-    /usr/include/x86_64-linux-gnu/sys/msg.h \
-    /usr/include/x86_64-linux-gnu/sys/mtio.h \
-    /usr/include/x86_64-linux-gnu/sys/personality.h \
-    /usr/include/x86_64-linux-gnu/sys/timeb.h \
-    /usr/include/x86_64-linux-gnu/sys/timex.h \
-    /usr/include/x86_64-linux-gnu/sys/vt.h \
-    toolchain
+cp -r --parents /usr/include toolchain
```

Comment:
This doesn't look safe to me. I'll need sometime to verify that it won't break certain builds.

Reply:
Because of  arm and x86 has too much diffs， may has some better way to copy specific files